ccfaq/__init__.py

```diff
@@ -41,8 +41,14 @@ async def on_message(message: discord.Message) -> None:
 @bot.event
 async def on_command(ctx):
     LOG.info('Fired %s by %s', ctx.command, ctx.author)
-
-
+    
+@bot.event
+async def on_raw_reaction_add(payload):
```

Comment:
I think it's worth doing `on_reaction_add` here instead. This does mean it only works on the last 1k messages, but that's acceptable given you only really want to use it on recent messages.

This means we can drop the `fetch_channel`/`fetch_message` calls, which are expensive to do for every single reaction.

Alternatively, we should at least move the `str(payload.emoji) == "\U0001f5d1\U0000fe0f"` to the top of this function, so we're only making those requests on relevant messages.